entry_types/scrolled/package/src/editor/config.js

```diff
@@ -9,6 +9,9 @@ import {EntryPreviewView} from './views/EntryPreviewView';
 import {SideBarRouter} from './routers/SideBarRouter';
 import {SideBarController} from './controllers/SideBarController';
 
+import {agent} from '../../../../../package/src/frontend/browser/Agent';
```

Comment:
Relative imports are for package internal things. `pageflow-scrolled` should neither need to know where the `pageflow` package lives on the file system nor its internal directory structure. Both is hard coded here. Since `pageflow/frontend` exports `browser` ([1](https://github.com/codevise/pageflow/blob/49e310350cabd8e78e5f349034521b44fb79812a/package/src/frontend/index.js#L7), [2](https://github.com/codevise/pageflow/blob/49e310350cabd8e78e5f349034521b44fb79812a/package/src/frontend/browser/index.js#L18), [3](https://github.com/codevise/pageflow/blob/49e310350cabd8e78e5f349034521b44fb79812a/package/src/frontend/browser/browser.js#L7)) and [assigns `browser.agent`](https://github.com/codevise/pageflow/blob/49e310350cabd8e78e5f349034521b44fb79812a/package/src/frontend/browser/index.js#L20), you should be able to do:

```
import {browser} from 'pageflow/scrolled`;
browser.agent.matchesSafari();
```